mava/systems/tf/maddpg/networks.py

```diff
@@ -127,8 +132,11 @@ def make_default_networks(
         # Get total number of action dimensions from action spec.
         num_dimensions = np.prod(agent_act_spec.shape, dtype=int)
 
-        # An optional network to process observations
-        observation_network = tf2_utils.to_sonnet_module(tf.identity)
+        # Create the shared observation network; here simply a state-less operation.
+        if observation_network is None:
+            observation_network = tf2_utils.to_sonnet_module(tf.identity)
+        else:
+            observation_network = observation_network
```

Comment:
We can remove this else since setting ` observation_network = observation_network` does nothing. I think I removed this for mappo's networks.

Reply:
Ah yes I remember lol